AeroDoc/app/src/main/res/values/strings.xml

```diff
@@ -22,6 +22,9 @@
     <string name="accept">Accept</string>
     <string name="dismiss">Dismiss</string>
     <string name="available">Available</string>
+    <string name="location_denied">Without permission there is no track moviment</string>
```

Comment:
Change the text to something like "Without permission the app can't track movement"